src/V3Width.cpp

```diff
@@ -2684,6 +2740,89 @@ class WidthVisitor final : public VNVisitor {
             nodep->v3error("Unknown built-in enum method " << nodep->prettyNameQ());
         }
     }
+    void methodCallWildcard(AstMethodCall* nodep, AstWildcardArrayDType* adtypep) {
+        AstCMethodHard* newp = nullptr;
+        if (nodep->name() == "num"  // function int num()
+            || nodep->name() == "size") {
```

Comment:
Need to test all these methods, copy the assoc method test and edit appropriately.

Reply:
Done.